src/main/java/edu/kis/vh/nursery/stack/StackInterface.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.vh.nursery.stack;
+
+public interface StackInterface {
+	public static final int EMPTY_STACK_VALUE = -1;

```

Comment:
Is it a correct approach to fixing errors showed by tests? Since (3.1.9) This value should remain 0.
Errors are still within the project.